setup.py

```diff
@@ -1,10 +1,31 @@
 import runpy
-from setuptools import setup
+import itertools
+from setuptools import setup, find_packages
 
 __version__ = runpy.run_path('heimdall/version.py')['__version__']
-setup(name='heimdall',
-      version=__version__,
-      author='Constantin Pape',
-      url='https://github.com/constantinpape/heimdall',
-      license='MIT',
-      scripts='scrips/view_contanier')
+
+requires = [
+    "napari",
+    "numpy"
+]
+
+extras = {
+    "hdf5": ["h5py"],
+    "zarr": ["zarr"],
```

Comment:
Shouldn't this be `z5py`?

Reply:
There didn't seem much point specifying z5py as a dependency in setup.py because setup.py wouldn't know how to install it, this was for the eventual zarr fallback, but it doesn't need to stay in for the time being. Once rust-pyn5's hdf5-like interface PR goes in that could also be a fallback for N5.

Reply:
That makes sense.